gemet/thesaurus/management/commands/check_spreadsheet.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+from gemet.thesaurus.models import Property
+
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+import re
+
+
+LABELS = {
+    'RT (GEMET)': True,
+    'RT (new)': False,
+    'BT (GEMET)': True,
+    'BT (new)': False,
+    'NT (GEMET)': True,
+    'NT (new)': False,
+}
+
+
+def check_term_database_existence(term):
+    return Property.objects.filter(value=term).exists()
+
+
+def split_text_into_terms(raw_text):
+    pattern = re.compile("[^a-zA-Z\d \-\\)\\(:]")
+    term_list = pattern.split(raw_text)
+    term_list = map(lambda term: term.strip(' \t\n;,.:').lower(), term_list)
+    term_list = filter(lambda term: term != '', term_list)
+    return term_list
+
+
+class Command(BaseCommand):
+    help = 'Check if spreadsheet terms are consistent.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('excel_file')
+
+    def check_term_existence(self, term, term_type, excel_cell_value,
+                             new_terms):
+        message = 'Error at cell {}: "{}"'.format(excel_cell_value, term)
+        if term_type:
+            if not check_term_database_existence(term):
```

Comment:
You can evaluate `Property.objects.filter(value=term).exists()` only once, before the `if` (you are evaluating it anyways on both `if` and `else`), and remove the function.